apache/trafficcontrol
# Allow Traffic Router binding DNS sockets to specific IP Address

## What does this PR (Pull Request) do?

- [x] This PR fixes #4361 

## Which Traffic Control components are affected by this PR?
- Documentation
- Traffic Router

## What is the best way to verify this PR?
- Set in dns.properties
```
dns.udp.host=127.0.0.1
dns.tcp.host=127.0.0.1
```
- Start Traffic Router
- Confirm listening IPs with `netstat -anlp | grep ":53"` 
- Verify with dig (`dig @127.0.0.1 <hostname>`) that TR can only be reached via the loopback IP and not through any other IP present on the server 


## The following criteria are ALL met by this PR


## Additional Information
- No UTs as there is not a good way to test applicationContext changes
